homeassistant/components/camera/synology.py

```diff
@@ -0,0 +1,131 @@
+"""
+Support for Synology Surveillance Station Cameras.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.synology/
+"""
+import logging
+
+import voluptuous as vol
+
+import requests
+
+from homeassistant.const import (
+    CONF_NAME, CONF_USERNAME, CONF_PASSWORD,
+    CONF_URL, CONF_CAMERA_NAME, CONF_STREAM_ID)
+from homeassistant.components.camera import (Camera, PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'Synology Camera'
+DEFAULT_STREAM_ID = '0'
+TIMEOUT = 10
+
+# pylint: disable=line-too-long

```

Comment:
Don't disable this one, just break up the line. Within parentheses strings will be auto concatenated.

```python
MJPEG_URL = ('{0}/webapi/SurveillanceStation/streaming.cgi?api='
             'SYNO.SurveillanceStation.Streaming&method=')
```